submissions/bmukha/frogger/app.js

```diff
@@ -0,0 +1,149 @@
+let highScore = localStorage.getItem('highScore') ?? 0;
+let currentScore = 0;
+const highHolder = document.querySelector('.high');
+const currentHolder = document.querySelector('.current');
+const infoHolder = document.querySelector('.info');
+
+const getRandomNumber = (min, max) =>
+  Math.floor(Math.random() * (max - min) + min);
+
+const directions = ['left', 'right'];
+
+const getRandomDirection = () =>
+  directions[Math.floor(Math.random() * directions.length)];
+
+canvas = {
+  width: 505,
+  height: 606,
+};
+
+class Enemy {
+  constructor(startY, mult) {
+    this.width = 101;
+    this.height = 171;
```

Comment:
I think these character width and height are true not only for enemy.
Also 101 is somehow relates to `505` above. Can we reduce number of constants and build some others using calculations?
As a peer developer tasked to amend this world (number of rows, columns, dimensions of tiles/characters) i'd like to change as few number as only possible.

P.S. Check magic numbers here and across entire codebase.